### PR TITLE
Add links redirecting users to the formal spec

### PR DESCRIPTION
**Pull request by binji**
See issue #1258.

### CODE REVIEW

File: BinaryEncoding.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Binary Encoding
 
+## **Note**: This document is no longer being updated. Please see the normative documentation [here](http://webassembly.github.io/spec/core/binary/index.html).
```
Comment (xtuc) [inline BinaryEncoding.md]: The bold styling doesn't seem to apply on a heading.
```suggestion
## Note: This document is no longer being updated. Please see the normative documentation [here](http://webassembly.github.io/spec/core/binary/index.html).
```

File: JS.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # JavaScript API
 
+## **Note**: This document is no longer being updated. Please see the normative documentation [here](http://webassembly.github.io/spec/js-api/index.html).
```
Comment (xtuc) [inline JS.md]:
```suggestion
## Note: This document is no longer being updated. Please see the normative documentation [here](http://webassembly.github.io/spec/js-api/index.html).
```

File: Semantics.md [hunk 3]
```diff
@@ -1,5 +1,7 @@
 # Semantics
 
+## **Note**: This document is no longer being updated. Please see the normative documentation [here](http://webassembly.github.io/spec/core/exec/index.html).
```
Comment (xtuc) [inline Semantics.md]:
```suggestion
## Note: This document is no longer being updated. Please see the normative documentation [here](http://webassembly.github.io/spec/core/exec/index.html).
```

File: TextFormat.md [hunk 4]
```diff
@@ -1,5 +1,7 @@
 # Text Format
 
+## **Note**: This document is no longer being updated. Please see the normative documentation [here](http://webassembly.github.io/spec/core/text/index.html).
```
Comment (xtuc) [inline TextFormat.md]:
```suggestion
## Note: This document is no longer being updated. Please see the normative documentation [here](http://webassembly.github.io/spec/core/text/index.html).
```

File: BinaryEncoding.md [hunk 5]
```diff
@@ -1,5 +1,7 @@
 # Binary Encoding
 
+## Note: This document is no longer being updated. Please see the normative documentation [here](http://webassembly.github.io/spec/core/binary/index.html).
```
Comment (Ms2ger) [inline BinaryEncoding.md]: Nit: "here" is not a great link text; better to link "normative documentation". LGTM otherwise.

Comment (binji) [reply BinaryEncoding.md]: Done

**Pull request merged**